lib/foodcritic/rules/fc078.rb

```diff
@@ -0,0 +1,135 @@
+rule "FC078", "Ensure cookbook shared under an OSI-approved open source license" do
+  tags %w{supermarket}
```

Comment:
I think the scenario is an org with a private supermarket. They want to ensure people use issues_url, but they don't care about open source licenses